slick-testkit/src/codegen/scala/slick/test/codegen/GenerateMainSources.scala

```diff
@@ -80,6 +80,17 @@ val  SimpleA = CustomTyping.SimpleA
       })
     },
     new UUIDConfig("CG10", StandardTestDBs.H2Mem, "H2Mem", Seq("/dbs/uuid-h2.sql")),
+    new Config("MySQL", StandardTestDBs.MySQL, "MySQL", Seq("/dbs/mysql.sql")){
```

Comment:
@trevorsibanda what's the version of your mysql? circleci successes because the test is not enabled.